### PR TITLE
Fix/code style

### PR DESCRIPTION
**Pull request by zenangst**

### CODE REVIEW

File: Source/FORMData.m [hunk 1]
```diff
@@ -618,7 +626,7 @@ - (NSArray *)showTargets:(NSArray *)targets
         if (target.type == FORMTargetTypeField) {
             [self fieldWithID:target.targetID includingHiddenFields:NO
                    completion:^(FORMField *field, NSIndexPath *indexPath) {
-                       if (field) shouldLookForField = NO;
+                       shouldLookForField = (!field);
```
Comment (NSElvis) [inline Source/FORMData.m]: I'm not sure if this works, for this might be better to `(field == nil)`

Comment (zenangst) [inline Source/FORMData.m]: Good point! I'll fix!

Comment (NSElvis) [inline Source/FORMData.m]: In order to make it no `(field != nil)`

Comment (zenangst) [inline Source/FORMData.m]: @NSElvis fixed in 9090718

**Comment by NSElvis** [comment]
:tada: !

**Pull request merged**